Program.cs

```diff
@@ -30,7 +30,7 @@
         throw new InvalidOperationException("Unable to find Raven cert password in configuration. This is stored in VS user secrets, Azure web app env vars, and LP.");
     }
     var certBytes = Convert.FromBase64String(certBase64);
-    o.Certificate = X509CertificateLoader.LoadPkcs12(certBytes, certPassword);
+    o.Certificate = new X509Certificate2(certBytes, certPassword);
```

Comment:
Don't use change this. The constructor you added here is obsolete in .NET 9. Please remove it.